### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by enola-dkfz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ To run a standalone Focus, you need at least one running [Samply.Beam.Proxy](htt
 You can compile and run this application via Cargo, however, we encourage the usage of the pre-compiled [docker images](https://hub.docker.com/r/samply/focus):
 
 ```bash
-docker run --rm -e BEAM_BASE_URL=http://localhost:8081 -e BLAZE_BASE_URL=http://localhost:8089/fhir -e PROXY_ID=proxy1.broker -e API_KEY=App1Secret -e APP_ID=app1 samply/focus:latest
+docker run --rm -e BEAM_BASE_URL=http://localhost:8081 -e BLAZE_BASE_URL=http://localhost:8089/fhir -e PROXY_ID=proxy1.broker -e API_KEY=App1Secret -e BEAM_APP_ID_LONG=app1 samply/focus:latest
```
Comment (TKussel) [inline README.md:17]: Nomenclature wise, `BEAM_APP_ID_LONG` should refer to `app1.broker.example.com`. For the short name, either `BEAM_APP_ID` or `BEAM_APP_ID_SHORT` would be better suited.

File: README.md [hunk 2]
```diff
@@ -26,17 +26,25 @@ BEAM_BASE_URL = "http://localhost:8081"
 BLAZE_BASE_URL = "http://localhost:8089/fhir"
 PROXY_ID = "proxy1.broker"
 API_KEY = "App1Secret"
-APP_ID = "app1"
+BEAM_APP_ID_LONG = "app1"
```
Comment (TKussel) [inline README.md:29]: Same as in line 17.

**Review by TKussel — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (enola-dkfz) [reply README.md]: done, there's only the long one

File: README.md (hunk 2 above)
Comment (enola-dkfz) [reply README.md]: done, there's only the long one

**Review by TKussel — APPROVED** [review]

**Pull request merged**